DopplerCustomDomain/Controllers/CustomDomainController.cs

```diff
@@ -43,7 +44,13 @@ public async Task<IActionResult> CreateCustomDomain([FromRoute] string domainNam
                 return new NotFoundObjectResult($"Cannot find the service called: {domainConfiguration.service}");
             }
 
-            await _customDomainProviderService.CreateCustomDomain(domainName, serviceName);
+            TraefikRuleTypeEnum ruleTypeEnum;
+            if (!Enum.TryParse(domainConfiguration.ruleType, out ruleTypeEnum))
+            {
+                ruleTypeEnum = TraefikRuleTypeEnum.HTTP;
```

Comment:
Maybe is better to return a bad request? this rule is now mandatory? if the client of this API doesn't send a rule, its good we decide for him?